plonetheme/barceloneta/theme/less/barceloneta-compiled.css

```diff
@@ -1323,6 +1323,58 @@ th {
     white-space: nowrap;
   }
 }
+table {
+  width: 100%;
```

Comment:
I think, this is a problematic declaration - it makes impossible to scale tables freely. Wouldn't be a ``min-width:1em`` or something be better?